src/lib/format.js

```diff
@@ -14,7 +14,7 @@ export function keysFromContext(ctx, keys, method) {
 export function valueFromContext(ctx, id, args) {
   const entity = ctx.messages.get(id);
 
-  if (!entity) {
+  if (entity === undefined) {
```

Comment:
Fixed in https://github.com/l20n/l20n.js/commit/c794525f3c1e453d7f8c295514f04fd78435a862